services/121-service/src/queues-registry/queues-registry.module.ts

```diff
@@ -24,6 +24,18 @@ import { AzureLogService } from '@121-service/src/shared/services/azure-log.serv
         duration: 1000, // per duration in milliseconds
       },
     }),
+    BullModule.registerQueue({
+      name: TransactionJobQueueNames.nedbank,
+      processors: [
+        {
+          path: 'src/transaction-job-processors/processors/transaction-job-safaricom.processor.ts',
+        },
+      ],
+      limiter: {
+        max: 5, // Max number of jobs processed
+        duration: 1000, // per duration in milliseconds
```

Comment:
Why 1000?

Reply:
Because 1000 ms is 1 second which is easy to understand for my brain ;)

Reply:
Why 1 second?